api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+    |       'description' => LocalizedString,
+    |   ]
+    |
+    */
+    'roles' => [
+        'guest' => [
+            'display_name' => [
+                'en' => 'Guest',
+                'fr' => 'Guest'
+            ],
+            'description' => [
+                'en' => 'These permissions are available to anyone, even not logged in.',
+                'fr' => 'Ces autorisations sont accessibles à tous, même aux personnes non connectées.'
+            ]
+        ],
+
+        'base_user' => [
+            'display_name' => [
+                'en' => 'Base User',
+                'fr' => 'Utilisateur de base'
+            ],
+            'description' => [
+                'en' => 'Available to any logged-in user.',
+                'fr' => 'Disponible pour tout utilisateur connecté.'
+            ]
+        ],
+
+        'applicant' => [
+            'display_name' => [
+                'en' => 'Applicant',
+                'fr' => 'Candidat'
+            ],
+            'description' => [
+                'en' => 'Can edit their own profile and apply to jobs.',
+                'fr' => 'Ils peuvent modifier leur propre profil et postuler à des emplois.'
+            ]
+        ],
+
+        'team_admin' => [
+            'display_name' => [
+                'en' => 'Team Admin',
+                'fr' => 'Team Admin'
+            ],
+            'description' => [
+                'en' => 'Can update their Team, add users to their Team, process applications, process requests and work on and publish pools',
+                'fr' => 'Ils peuvent mettre à jour leur équipe, ajouter des utilisateurs à leur équipe, traiter des demandes, travailler sur des pools et les publier.'
+            ]
+        ],
+
+        'super_admin' => [
+            'display_name' => [
+                'en' => 'Super Admin',
+                'fr' => 'Super Admin'
+            ],
+            'description' => [
+                'en' => 'Makes teams, assigns roles to other users (including assigning users to orgs), manages lists of business data, and has the extraordinary ability to edit or delete other users.',
+                'fr' => 'Il crée des équipes, attribue des rôles à d\'autres utilisateurs (y compris l\'attribution d\'utilisateurs à des organisations), gère des listes de données commerciales et a la capacité extraordinaire de modifier ou de supprimer d\'autres utilisateurs.'
+            ]
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Seeders
+    |--------------------------------------------------------------------------
+    |
+    | A map of the values to seed with the shape:
+    |
+    |   'role_name' => [
+    |       'resource' => [
+    |           'scope' => [ 'actions' ]
+    |       ]
+    |   ]
+    |
+    */
+    'seeders' => [
+        'guest' => [
+            'clss' => [
+                'a' => ['v']
+            ],
```

Comment:
Let's use the full names here, instead of the short keys. Like:
```
            'classification' => [
                'any' => ['view']
            ],
```
 I think it will be a lot more readable. And should be easy now with a find-replace.

Reply:
Updated the maps to work more like enums for readability in [use full words instead of abbrs for stubs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5560/commits/d3fd102751dbe14e9927ac5bf4a83b231d04f103)